src/app/dataset-description/dataset-description.component.html

```diff
@@ -17,10 +17,15 @@
 
   <div *ngIf="editMode">
     <div class="editor">
-      <angular-markdown-editor textareaId="editor1" [options]="editorOptions" [(ngModel)]="editorText">
+      <angular-markdown-editor
+        textareaId="editor1"
+        [options]="editorOptions"
+        [(ngModel)]="editorText"
+        (onChange)="togglePreview()"
+        (onShow)="disablePreviewOnLoad(!dataset.description)">
       </angular-markdown-editor>
-      <button (click)="save()" class="btn btn-success" title="Save (ctrl+enter)">Save</button>
-      <button (click)="close()" class="btn" style="margin-left: 8px" tutle="Close">Close</button>
+      <button (click)="save()" id="save-btn" class="btn btn-success" title="Save (ctrl+enter)">Save</button>
+      <button (click)="close()" id="close-btn" class="btn" title="Close">Close</button>
```

Comment:
I'd steer away from using "btn" (as a term) as it's commonly used in bootstrap classes
rename "save-btn" to "save-button"
rename "close-btn" to "close-button"